### PR TITLE
murdock: make available boards instance dependent

### PR DESCRIPTION
**Pull request by kaspar030**
### Testing procedure

This PR is the next step in migrating to the new CI frontend.
It basically configures the available pifleet boards depending on the actual CI instance.
This also disables the default test board collection for our current CI, ci-prod will take over once this is merged.

### CODE REVIEW

File: .murdock [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+export APPS="examples/hello-world tests/unittests"
+
+# this configures boards that are available via pifleet
+case "${CI_MURDOCK_PROJECT}" in
+    riot)
+        : ${TEST_BOARDS_AVAILABLE:="esp32-wroom-32 samr21-xpro"}
+        ;;
+    riot-staging)
+        : ${TEST_BOARDS_AVAILABLE:=""}
+        ;;
```
Comment (miri64) [inline .murdock:15]: Is there a plan to add boards to that instance in the future or why is there a separate branch for `riot-staging`?

Comment (miri64) [reply .murdock:15]: (it is identical to the default branch, that was why I was asking)

**Comment by kaspar030** [comment]
Some test runs. 

This is on ci-prod with both boards in the list:
https://ci-prod.riot-os.org/details/4268dafc89b1445ba79f68ed9fe5a973/tests (both got tested)

And this is with only samr21-xpro in the list:
https://ci-prod.riot-os.org/details/f1043c98c310485e924cf3e81e0f9875/tests

So this works as expected.

File: .murdock (hunk 1 above)
Comment (kaspar030) [reply .murdock:15]: yes... we might want to test things with boards there I guess.

That makes me thinking if this should be configured here at all.

Comment (kaspar030) [reply .murdock:15]: ok, so to pass that info from the new frontend, that would need a restart, stopping current builds and clearing the queue.

I'd like to keep the riot-staging even if it is identical, so we see right away where and now to add board(s) there.

**Review by maribu — APPROVED** [review]

**Pull request merged**